### DISCUSSION TITLE
Notifications essential impovement

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#15852] [Feature improvement]
Notification field Recipients, doesn't contains "Assigned to" but record owner, and list of single users/roles. It bringing significant disadvantage in the scenario when record is created via API. Adding to Recipients list "Assigned to" will fix  above.

**Comment by JohnMaverick** [comment]
noticed that too. Recipients should have Assigned to... tried way around by creating entity record "Notification" but it not showing on the list of modules in create entity workflow action

**Reply by TonyM15** [reply]
thumb up here too
